<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSearchUser</class>
 <widget class="QWidget" name="FormSearchUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbCedula">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrese Cedula:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNombre">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>73</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbApellido">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellido">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCedula">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbTipo2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
